src/uyuni_health_check/main.py

```diff
@@ -568,6 +520,8 @@ def run_loki(server):
                 f"{promtail_cfg}:/etc/promtail/config.yml",
                 "-v",
                 "/var/log/:/var/log/",
+                "-v",
+                f"{supportconfig_path}:{supportconfig_path}",
```

Comment:
I suppose that you need to have a different promtail config for the supportconfig and running cases with such a mount, right?

Here I mounted the files where those are expected on a regular host:
```diff
+        volumes = ["-v", "/var/log:/var/log"]
+        if supportconfig:
+            log_map = {
+                "rhn-logs/rhn": "rhn",
+                "httpd-logs/apache2": "apache2",
+                "salt-logs/salt": "salt",
+                "tomcat-logs/tomcat": "tomcat",
+                "cobbler-logs": "cobbler",
+                "audit-log": "audit",
+            }
+            volumes = []
+            for host, mount in log_map.items():
+                volumes += ["-v", "{}/spacewalk-debug/{}:/var/log/{}".format(supportconfig, host, mount)]
+
+            # Special handling for pgsql logs
+            volumes += ["-v", "{}/spacewalk-debug/database:/var/lib/pgsql/data/log".format(supportconfig)]
```

I'm not sure if that could help you.

Reply:
I did it differently, as I mount the root of the supportconfig, instead of the different files.

For the "promtail" configuration, now we use a jinja2 template file which is common for both cases (live server / supportconfig) but we render it passing the right paths to the files depending on the scenario.